posthog/src/main/java/com/posthog/java/PostHog.java

```diff
@@ -31,15 +31,26 @@ public Builder host(String host) {
             return this;
         }
 
+        public PostHog build() {
+            Sender sender = new HttpSender.Builder(apiKey).host(host).build();
+            this.queueManager = new QueueManager.Builder(sender).build();
+            return new PostHog(this);
+        }
+    }
+
+    public static class BuilderWithCustomQueueManager extends BuilderBase {
```

Comment:
Need this for testing, didn't want to start validating that people don't try to provide queuemanager and host separately as that doesn't work. Maybe there's a nicer way 🤷‍♀️